src/js/utils/spec-helpers/ajax-helpers.js

```diff
@@ -1,9 +1,20 @@
 define(['sinon', 'underscore', 'URI'], function(sinon, _, URI) {
     'use strict';
 
-    var fakeRequests, expectRequest, expectJsonRequest, expectPostRequest, expectRequestURL,
+    var XML_HTTP_READY_STATES, fakeServer, createFakeRequests, withFakeRequests, fakeRequests, currentRequest,
```

Comment:
Nit: perhaps rename `XML_HTTP_READY_STATES` to `XHR_READY_STATES` because it's a bit shorter, and it masks XML as "X" :). I think the whole "XML" part of "XHR" can be misleading these days when XHRs are used for all kinds of content-types.